lib/stripe/payment_methods/card.ex

```diff
@@ -46,7 +46,9 @@ defmodule Stripe.Card do
           fingerprint: String.t() | nil,
           funding: String.t(),
           last4: String.t(),
-          metadata: Stripe.Types.metadata(),
+          metadata: %{
+            optional(String.t()) => String.t()
+          },
```

Comment:
See comment above. Again, sorry if there's a conversation about this somewhere which I forgot about.